rotkehlchen/tests/unit/test_calendar_reminders.py

```diff
@@ -260,3 +262,126 @@ def mock_requests_get(url: str, timeout: int = 0, headers: dict | None = None):
+        counterparty=bridge_event.counterparty,
+        auto_delete=True,
+        blockchain=arbitrum_one_inquirer.chain_id.to_blockchain(),
+        address=arbitrum_one_accounts[0],
+        color=BRIDGE_CALENDAR_COLOR,
+    )
+
+    # one reminder is created at the time of the calendar entry
+    reminders = calendar_db.query_reminder_entry(event_id=calendar_entry.identifier)['entries']
+    assert len(reminders) == 1
+    assert reminders[0].event_id == calendar_entry.identifier
+    assert reminders[0].secs_before == 0
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.freeze_time('2023-01-09 08:34:00 GMT')
+@pytest.mark.parametrize('optimism_accounts', [[
+    '0xE232E72983E329757F02292322296f5B96dAfC8F',
+]])
+def test_l2_optimism_bridge_claim_reminder(optimism_accounts, optimism_inquirer, database):
+    evmhash = deserialize_evm_tx_hash('0xe2111cddcd42c8214770c7a3270490c31663cd8b4b20b3fc27018ca3ce7a3979')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=optimism_inquirer, tx_hash=evmhash)
+
+    calendar_db = DBCalendar(database)
+    all_calendar_entries = calendar_db.query_calendar_entry(CalendarFilterQuery.make())
+    assert all_calendar_entries['entries_total'] == 0
+
+    reminder_creator = CalendarReminderCreator(database=database, current_ts=ts_now())
+    reminder_creator.maybe_create_l2_bridging_reminder()
+
+    new_calendar_entries = calendar_db.query_calendar_entry(CalendarFilterQuery.make())
+    assert new_calendar_entries['entries_found'] == 1
+
+    calendar_entry = new_calendar_entries['entries'][0]
+    bridge_event = events[-1]
+    assert calendar_entry == CalendarEntry(
+        identifier=1,
+        name=f'Claim {bridge_event.balance.amount} {bridge_event.asset.symbol_or_name()} bridge deposit on Ethereum',  # noqa: E501
+        timestamp=ts_ms_to_sec(TimestampMS(bridge_event.timestamp + WEEK_IN_SECONDS)),
+        description=f'Bridge deposit of {bridge_event.balance.amount} {bridge_event.asset.symbol_or_name()} is ready to claim on Ethereum',  # noqa: E501
+        counterparty=bridge_event.counterparty,
+        auto_delete=True,
+        blockchain=optimism_inquirer.chain_id.to_blockchain(),
+        address=optimism_accounts[0],
+        color=BRIDGE_CALENDAR_COLOR,
+    )
+
+    # one reminder is created at the time of the calendar entry
+    reminders = calendar_db.query_reminder_entry(event_id=calendar_entry.identifier)['entries']
+    assert len(reminders) == 1
+    assert reminders[0].event_id == calendar_entry.identifier
+    assert reminders[0].secs_before == 0
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.freeze_time('2024-10-18 07:24:00 GMT')
+@pytest.mark.parametrize('base_accounts', [[
+    '0x6730b1Df17E50217777EeE475E34815964e3BFb2',
+]])
+def test_l2_base_bridge_claim_reminder(base_accounts, base_inquirer, database):
```

Comment:
it seems that you can abstract those 3 tests into 1 with a for loop, don't you think? You would have 3 inquirers loaded and query 3 chains but that is fine